### ISSUE TITLE
Feature Request: Motion blur

### ISSUE THREAD

**Issue by HydrogenC** [o3de/o3de#18227]
**Is your feature request related to a problem? Please describe.**
I cannot find a post processing component that adds the motion blur effect, nor could it be founded in the docs or source code. 

**Describe the solution you'd like**
Add motion blur as a post-processing effect. I remember there used to be motion blur in lumberyard, and it is indeed fairly common to use motion blur in 3A games. 

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by moudgils** [comment]
yeah. This specific feature is currently not implemented within the engine although should be easy to leverage from existing literature in the industry. Lumberyard had a decent implementation of MB so we could just port that. In order to add proper MB implementation we will need to take both the camera blur and object blur into account. First step should be a RFC if anyone is inclined to take this on.
